CONTRIBUTING.md

```diff
@@ -0,0 +1,81 @@
+# How to contribute
+
+WIP
+
+## Clone the repo
+
+```
+> git clone https://github.com/cockroachdb/ui/
+```
+
+The UI repo is a monorepo managed by [Lerna](https://lerna.js.org/). Once you clone the repo, be sure to [bootstrap](). _In the following examples, I'm using `npx`. If you do this very often you may consider installing Lerna globally (`npm i -g lerna`)._
+
+```
+> cd path/to/ui
+> npx lerna bootstrap
+```
+
+## Branch and develop
```

Comment:
I think we should follow the same pattern we use with all of our other apps (i.e., fork --> branch --> PR). Is that what this guide is describing?

The "Where to push" section of the [CRDB contributing guidelines](https://wiki.crdb.io/wiki/spaces/CRDB/pages/181633464/Your+first+CockroachDB+PR) has a description of the CRDB process.